### PR TITLE
feat: added image and code block caption with example

### PR DESCRIPTION
**Pull request by WiebkeFr**
pr contains:
- components for image and code block caption with changeable design
- example for both in a blog article

to test: [https://satellytescommain-featimagecaptioninmarkdown.gtsb.io/blog/github-billing-dashboard/](https://satellytescommain-featimagecaptioninmarkdown.gtsb.io/blog/github-billing-dashboard/)

technical notes:
- the caption of images is styled with css: [https://thesynack.com/posts/markdown-captions/#targeting-captions-with-css](https://thesynack.com/posts/markdown-captions/#targeting-captions-with-css)
- the code block caption is done with `<figurecaption>` [https://developer.mozilla.org/en-US/docs/Web/HTML/Element/figure](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/figure)
- image captions can be done with `<figurecaption>` as well, but I would say the first option is easier to write (as it does not require additional html)

### CODE REVIEW

File: blog-posts/howto-blog-post.md [hunk 1]
```diff
@@ -67,7 +85,7 @@ determine the expected syntax. If you omit
 
 ```
 ~~~javascript
-cosnt myJavaScrip = true;
```
Comment (georgiee) [inline blog-posts/howto-blog-post.md:70]: 👍

File: blog-posts/howto-blog-post.md [hunk 2]
```diff
@@ -53,7 +53,25 @@ Here a 4MB picture that should get prepared by gatsby properly.
...
-<a href='https://www.freepik.com/vectors/star'>Star vector created by upklyak</a>
+*[Star vector created by upklyak](https://www.freepik.com/vectors/star)*
+
+#### Image caption
+
+To maintain a valid HTML image structure and an easily readable markdown-file, the caption for the image can be added like this:
+```
+![alt-text](./image-url)
+*Picture 1: This is an example caption.*
+```
```
Comment (georgiee) [inline blog-posts/howto-blog-post.md:64]: this is different to the billing dashboard example where the text is just added as a paragraph?

File: blog-posts/github-billing-dashboard.md [hunk 3]
```diff
@@ -27,7 +27,9 @@ The following screenshot shows all the billing information you get from the GitH
 ![screenshot of the billing settings in a Github account](images/gh-billing-dashboard-screenshot-of-gh-billing-settings.png)
 
 And this is what you get from the GitHub Billing Dashboard. Costs per service per month with historic data:
+
 ![screenshot of the Github billing dashboard with some charts](images/gh-billing-dashboard-screenshot-of-gh-dashboard.png)
```
Comment (georgiee) [inline blog-posts/github-billing-dashboard.md:31]: I don't see an caption being added?

File: blog-posts/howto-blog-post.md [hunk 4]
```diff
@@ -166,6 +184,16 @@ footer p {
...
+#### Code Block Caption
+
+A code block caption can be added with:
+```html
+<figure>
+  
+  // Code Block
+  <figcaption>Code Block 1: This is an example for a caption.</figcaption>
+</figure>
+```
```
Comment (georgiee) [inline blog-posts/howto-blog-post.md:196]: I don't see an example for this in the markdown being rendered which would make sense here, wouldn't it ?

**Review by georgiee — CHANGES_REQUESTED** [review]
I see the feature being provide but not really used or demonstrated. I think we should double check if this change is ok.

**Comment by feedm3** [comment]
It looks like this PR is not working currently.

The idea here is that we provide captions for images and code blocks. I outlined it here https://github.com/satellytes/satellytes.com/issues/102

We can use Notion as guidance. They support this great feature really well.

**Comment by georgiee** [comment]
Thanks for the feedback. The PR didn't make it clear what to expect and taking Notion as the guidance makes it much easier.

**Comment by georgiee** [comment]
This is outdated and covered by these issues for the updated/aligned desing:

+ https://github.com/satellytes/satellytes.com/issues/179
+ https://github.com/satellytes/satellytes.com/issues/175

**Pull request closed without merging**